demo/apisix/conf/apisix.yaml

```diff
@@ -6,6 +6,82 @@ routes:
       nodes:
       type: roundrobin
     plugin_config_id: 1
+  -
+    id: jmap
+    uri: /jmap
+    service_id: jmap_service
+    methods:
+      - POST
+  -
+    id: jmap_session
+    uri: /jmap/session
+    service_id: jmap_service
+    methods:
+      - GET
+  -
+    id: download
+    uri: /download/*
+    service_id: jmap_service
+    methods:
+      - GET
+      - OPTIONS
+  -
+    id: upload
+    uri: /upload
+    service_id: jmap_service
+    methods:
+      - POST
+      - OPTIONS
+  -
+    id: upload
+    uri: /upload/*
+    service_id: jmap_service
+    methods:
+      - POST
+      - OPTIONS
+  -
+    id: web_known_finger
+    uri: /.well-known/webfinger
+    service_id: tmail_no_auth_service
+    methods:
+      - GET
+  -
+    id: web_known_jmap
+    uri: /.well-known/jmap
+    service_id: jmap_service
+    methods:
+      - GET
+  -
+    id: web_known_linagora_ecosystem
+    uri: /.well-known/linagora-ecosystem
+    service_id: jmap_service
+    methods:
+      - GET
+
+
+services:
+  -
+    id: jmap_service
```

Comment:
e.g `jmap_service_oidc`?

(In the future we can add JMAP basic auth service e.g `jmap_service_basic_auth`)